### DISCUSSION TITLE
Salons avancés

### DISCUSSION THREAD

**Discussion by KN4MSH** [marcbat/svxlinkmanager#64] [Q&A]
We need some documentation related to this future. How are we going to use this capability ?

**Comment by marcbat** [comment]
Hello,
You can use this feature to configure a custom connection to a reflector. All parameters supported by Svxlink 19.09.1 is available. 

“Nom du salon”: is the name of your custom configuration. After you save the new configuration, you can select this in “Accueil” page, in the drop-down control. 

“Annonce vocal”: A wav file play by svxlink when you connect to the “Salon” (Room maybe in English?) 

DTMF: when you compose this DTMF on your TX, svxlink run this configuration and connect to the reflector. 

“Salon par défaut”: When is check, this configuration become de default configuration. 

“Temporisé”: If is check, after an inactivity, svxlink return to the default Room. 

The only required tab is “Svxlink”.
You can find the documentation on this website http://www.svxlink.org/doc/man/man5/svxlink.conf.5.html.

Il you want make a try, you can use the following example to connect “International Room”.


```ini
[GLOBAL]
LOGICS=SimplexLogic,ReflectorLogic
CFG_DIR=svxlink.d
TIMESTAMP_FORMAT=% c
CARD_SAMPLE_RATE=16000
CARD_CHANNELS=1
LINKS=ALLlink

[SimplexLogic]
TYPE=Simplex
RX=Rx1
TX=Tx1
MODULES=ModuleHelp,ModuleMetarInfo,ModulePropagationMonitor
CALLSIGN=KN4MSH
SHORT_IDENT_INTERVAL=15
LONG_IDENT_INTERVAL=60
IDENT_ONLY_AFTER_TX=10
EXEC_CMD_ON_SQL_CLOSE=500
EVENT_HANDLER=/usr/share/svxlink/events.tcl
DEFAULT_LANG=fr_FR
RGR_SOUND_ALWAYS=1
RGR_SOUND_DELAY=0
REPORT_CTCSS=71.9
TX_CTCSS=ALWAYS
MACROS=Macros
FX_GAIN_NORMAL=0
FX_GAIN_LOW=-12
ACTIVATE_MODULE_ON_LONG_CMD=10:PropagationMonitor
MUTE_RX_ON_TX=1
DTMF_CTRL_PTY=/tmp/dtmf_uhf

[ALLlink]
CONNECT_LOGICS=SimplexLogic:434MHZ:945,ReflectorLogic
DEFAULT_ACTIVE=1
TIMEOUT=0

[Rx1]
TYPE=Local
AUDIO_DEV=alsa:plughw:0
AUDIO_CHANNEL=0
SQL_DET=GPIO
SQL_START_DELAY=500
SQL_DELAY=100
SQL_HANGTIME=20
SQL_EXTENDED_HANGTIME=1000
SQL_EXTENDED_HANGTIME_THRESH=13
SQL_TIMEOUT=600
VOX_FILTER_DEPTH=300
VOX_THRESH=1000
CTCSS_MODE=2
CTCSS_FQ=71.9
CTCSS_SNR_OFFSET=0
CTCSS_OPEN_THRESH=15
CTCSS_CLOSE_THRESH=9
CTCSS_BPF_LOW=60
CTCSS_BPF_HIGH=260
GPIO_PATH=/sys/class/gpio
GPIO_SQL_PIN=gpio10
DEEMPHASIS=0
SQL_TAIL_ELIM=0
PREAMP=-4
PEAK_METER=1
DTMF_DEC_TYPE=INTERNAL
DTMF_MUTING=1
DTMF_HANGTIME=40
1750_MUTING=1

[Tx1]
TYPE=Local
AUDIO_DEV=alsa:plughw:0
AUDIO_CHANNEL=0
PTT_TYPE=GPIO
GPIO_PATH=/sys/class/gpio
PTT_PIN=gpio7
TIMEOUT=300
TX_DELAY=900
PREAMP=0
CTCSS_FQ=Pas de tone
CTCSS_LEVEL=9
PREEMPHASIS=0
DTMF_TONE_LENGTH=100
DTMF_TONE_SPACING=50
DTMF_DIGIT_PWR=-15

[ReflectorLogic]
TYPE=Reflector
AUDIO_CODEC=OPUS
JITTER_BUFFER_DELAY=2
CALLSIGN=(US) KN4MSH H
HOST=rrf3.f5nlg.ovh
AUTH_KEY=Magnifique123456789!
PORT=5302
```

**Reply by KN4MSH** [reply]
Thank you for your guide. I am just testing the configuration script.

**Comment by marcbat** [comment]
In the version 3.0.0 and after, you can configure and host your own reflector. But you must use the version 3.0.0 of the image.

**Reply by KN4MSH** [reply]
Hi, again. I just found some errors, when SvxlinkManager changes the rooms(salons):

Starting logic: SimplexLogic
Nov 13 12:28:23 SvxlinkManager svxlinkmanager[1085]: SvxlinkManager.Service.SvxLinkService[0] Loading RX: Rx1
Nov 13 12:28:23 SvxlinkManager svxlinkmanager[1085]: SvxlinkManager.Service.SvxLinkService[0] Loading TX: Tx1
Nov 13 12:28:23 SvxlinkManager svxlinkmanager[1085]: SvxlinkManager.Service.SvxLinkService[0] Loading module "ModuleHelp" into logic "SimplexLogic"
Nov 13 12:28:23 SvxlinkManager svxlinkmanager[1085]: SvxlinkManager.Service.SvxLinkService[0] *** ERROR: Failed to load module ModuleHelp into logic SimplexLogic: /usr/lib/arm-linux-gnueabihf/svxlink/ModuleModuleHelp.so: cannot open shared object file: No such file or directory
Nov 13 12:28:23 SvxlinkManager svxlinkmanager[1085]: SvxlinkManager.Service.SvxLinkService[0] Loading module "ModuleMetarInfo" into logic "SimplexLogic"
Nov 13 12:28:23 SvxlinkManager svxlinkmanager[1085]: SvxlinkManager.Service.SvxLinkService[0] *** ERROR: Failed to load module ModuleMetarInfo into logic SimplexLogic: /usr/lib/arm-linux-gnueabihf/svxlink/ModuleModuleMetarInfo.so: cannot open shared object file: No such file or directory
Nov 13 12:28:23 SvxlinkManager svxlinkmanager[1085]: SvxlinkManager.Service.SvxLinkService[0] Loading module "ModulePropagationMonitor" into logic "SimplexLogic"
Nov 13 12:28:23 SvxlinkManager svxlinkmanager[1085]: SvxlinkManager.Service.SvxLinkService[0] *** ERROR: Failed to load module ModulePropagationMonitor into logic SimplexLogic: /usr/lib/arm-linux-gnueabihf/svxlink/ModuleModulePropagationMonitor.so: cannot open shared object file: No such file or directory
Nov 13 12:28:23 SvxlinkManager svxlinkmanager[1085]: SvxlinkManager.Service.SvxLinkService[0] SimplexLogic: Event handler script successfully loaded.
Nov 13 12:28:23 SvxlinkManager svxlinkmanager[1085]: SvxlinkManager.Service.SvxLinkService[0] Name.wav file exists!
